src/services/deleteTokenLogoutService.js

```diff
@@ -18,12 +20,14 @@ angular.module('openshiftCommonServices')
         }
 
         // Lazily get the data service. Can't explicitly depend on it or we get circular dependencies.
```

Comment:
do you mind updating the comment since it applies to both services?

Reply:
Sure thing!